pyproject.toml

```diff
@@ -39,6 +39,7 @@ extension-pkg-allow-list = [
     "av.stream",
     "ciso8601",
     "cv2",
+    "evdev._ecodes",
```

Comment:
I'm not we should add this since the manual for pylint states:

`Be aware though that using this flag means that extensions are loaded into the active Python interpreter and may run arbitrary code, which you may not want.`

I'll ask for a second opinion on this though.

Reply:
I think I could remove the need for this by just using not using the [evdev.ecodes](https://github.com/gvalkov/python-evdev/blob/master/evdev/ecodes.py#L86) consts (using their values instead). Only using it in one place anyway.